src/viewProviders/base.test.ts

```diff
@@ -51,18 +53,23 @@ class TestViewProvider extends BaseViewProvider<FlinkComputePool, FlinkStatement
   }
 
   testEventEmitter: EventEmitter<void> = new EventEmitter<void>();
-  handleCustomListenerCallback() {}
+  handleCustomListenerCallback() {
+    this.logger.info("Custom event listener callback executed.");
```

Comment:
I might be missing it, but was should there be an assertion that `logger.info` was called during the custom event handler testing?

Reply:
All good, I was mainly curious if there was a callpoint in response to that logger bit being added (since we didn't previously test the innards of custom callbacks)